modules/updater.py

```diff
@@ -141,7 +146,7 @@ def rootfsOverlay(self):
             if not os.path.exists(src_full_path):
                 log.warn(src_full_path + " was not found in your current mounted rootfs. Can't overlay.")
                 continue
-            if not safeCopy(src_full_path, self.tempRootMountpoint + dst):
+            if not safeCopy(src_full_path, self.tempRootMountpoint + dst, sync=False):
```

Comment:
I don't think this is a necessary change especially since it introduces a subtle way of breaking the updater which is the power cutting after having switched the bootloader but before these writes have been persisted. Since performance isn't an issue I suggest keeping all the operations safe